docs/develop/parent-to-dialog.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Passing data and messages to a dialog box from its host page
+description: 'Learn how to pass data to a dialog from the host page'
+ms.date: 03/11/2020
+localization_priority: Normal
+---
+
+# Passing data and messages to a dialog box from its host page
+
+Microsoft is now making available for preview new APIs for messaging from the host page (defined [here](dialog-api-in-office-add-ins.md#open-a-dialog-box-from-a-host-page)) to the dialog box.
```

Comment:
Can we shift the tone of this from a preview announcement to more of a feature description?